Source/WebCore/css/CSSProperties.json

```diff
@@ -3593,6 +3593,25 @@
                 "url": "https://www.w3.org/TR/SVG11/text.html#DominantBaselineProperty"
             }
         },
+        "dynamic-range-limit": {
+            "inherited": true,
+            "values": [
+                "standard",
+                "constrained-high",
+                "high"
+            ],
+            "codegen-properties": {
+                "settings-flag": "cssDynamicRangeLimitPropertyEnabled",
```

Comment:
Is this sufficient to gate the parsing given the code is not auto-generated? Does `consumeDynamicRangeLimit` also need to check this?

Reply:
Looks like it's being taken care by https://searchfox.org/wubkat/source/__GENERATED__/WebCore/DerivedSources/CSSPropertyParsing.cpp#4802